lms/djangoapps/class_dashboard/tests/test_dashboard_data.py

```diff
@@ -98,7 +111,7 @@ def setUp(self):
             for j, user in enumerate(self.users):
                 StudentModuleFactory.create(
                     grade=1 if i < j else 0,
-                    max_grade=1 if i < j else 0.5,
+                    max_grade=1,
```

Comment:
Now I've added a content lib problem the number of 'problems' doesn't match the number of users. The test wasn't written that well several years ago.